config/step/ot_finngen_finemapping_ingestion.yaml

```
@@ -0,0 +1,7 @@
+defaults:
+  - finngen_finemapping_ingestion
+
+finngen_finemapping_results_url: ${datasets.finngen_finemapping_results_url}
+finngen_finemapping_summaries_url: ${datasets.finngen_finemapping_summaries_url}
```

I think this is outdated
```suggestion
finngen_finemapping_summaries_url: ${datasets.finngen_finemapping_summaries_path}
```